monkestation/code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -19,3 +19,36 @@
 	gold_core_spawnable = NO_SPAWN
 	unique_pet = TRUE
 	speak = list("barks!", "woofs!", "borks!", "yips!", "burks!")
+
+/mob/living/simple_animal/pet/dog/corgi/borgi
+	name = "E-N"
+	real_name = "E-N"
+	desc = "It's a borgi."
+	icon = 'monkestation/icons/mob/pets.dmi'
+	icon_state = "borgi"
+	icon_living = "borgi"
+	icon_dead = "borgi_dead"
+	var/emagged = FALSE
+	minbodytemp = 0
+	loot = list(/obj/effect/decal/cleanable/robot_debris)
+	del_on_death = TRUE
+	deathmessage = "blows apart!"
+	animal_species = /mob/living/simple_animal/pet/dog/corgi/borgi
+	nofur = TRUE
+
+/mob/living/simple_animal/pet/dog/corgi/borgi/emag_act(user as mob)
+	if(!emagged)
+		emagged = TRUE
+		visible_message("<span class='warning'>[user] swipes a card through [src].</span>", "<span class='notice'>You overload [src]s internal reactor.</span>")
+		addtimer(CALLBACK(src, .proc/explode), 1000)
```

Comment:
seconds define

Reply:
seconds define?

Reply:
what do you mean by that?

Reply:
Make it into a seconds define (`100 SECONDS`) instead of 1000 (which is 1000 deci seconds)